packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1188,10 +1161,6 @@ export class ContainerRuntime
 
 	private _idCompressor: (IIdCompressor & IIdCompressorCore) | undefined;
 
-	// We accumulate Id compressor Ops while Id compressor is not loaded yet (only for "delayed" mode)
-	// Once it loads, it will process all such ops and we will stop accumulating further ops - ops will be processes as they come in.
-	private pendingIdCompressorOps: IdCreationRange[] = [];
-
```

Comment:
The new behavior for "delayed" is to create the IdCompressor, but wait for the migration to "on" before we use it.  

Consequently, we should no longer need pendingIdCompressorOps.